SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs

```diff
@@ -133,4 +141,14 @@ public partial class SettingsPageViewModel : ViewModelBase
         _localizationService.Translate(value);
         Config.CurrentLanguageSelected = value.Name;
     }
+
+    partial void OnCurrentAccentColorChanged(Color? value)
+    {
+        if (faTheme == null) faTheme = Application.Current?.Styles.OfType<FluentAvaloniaTheme>().FirstOrDefault();
```

Comment:
I've never observed this trigger, Is this needed?

Reply:
> I've never observed this trigger, Is this needed?

For the theme specificly yes, I dig for one hour trying to get the FluentAvaloniaTheme. Lucky me, I found it.